### ISSUE TITLE
Pod has unbound immediate PersistentVolumeClaims

### ISSUE THREAD

**Issue by simonxxz** [meilisearch/meilisearch-kubernetes#45]
Hi guys! I installed meilisearch on my k8s cluster and it's pending forever. It shows 'failedscheduling‘ for 'Pod has unbound immediate PersistentVolumeClaims', even though I enabled pvc persistence in values.yaml. Here is the code:
persistence:
  type: pvc
  enabled: true
  storageClassName: default
  accessMode: ReadWriteOnce
  size: 10Gi
  finalizers:
    - kubernetes.io/pvc-protection
   
 Any help would be really appreciated!

**Comment by curquiza** [comment]
Hello @simonxxz!
@eskombro is unavailable this week and will help you on this during next week 🙂

**Comment by simonxxz** [comment]
> Hello @simonxxz!
> @eskombro is unavailable this week and will help you on this during next week

Hi curquiza! Thank you for that information! I actually solved this pvc problem by changing the 'storageClassName' to 'storageClass'. Don't know it was my typo or it's from the older versions. Tho I ran into another problem where meili stores documents and deals with searching requests at the same time; I keep getting 'http error 408' on the webUI and the meili pod restarts spontaneously. Maybe I should open another issue to address this?
